tests/business.test.ts

```diff
@@ -176,7 +176,9 @@ import { Marqeta } from '../src';
   console.log('testing list Business transitions by token Id...')
   if (listItem?.token) {
     let listTrans
-    listTrans = await client.business.listTransition(listItem?.token)
+    listTrans = await client.business.listTransition(
+      { token: listItem?.token }
+    )
```

Comment:
This isn't bad, but the way you have built the initial call, you can't allow any excess parameters in the argument. But it's _possible_ to imagine building up the call in the `listTransition()` call, and then you could pass in `listItem` as the argument, and it would simple "pick out" the values it cares about.